react/src/Analyses/index.tsx

```diff
@@ -186,8 +186,9 @@ export default function Analyses() {
         "analysisTableSortOrder"
     );
 
-    function changeAnalysisState(newState: PipelineStatus) {
-        return _changeStateForSelectedRows(activeRows, analysisUpdateMutation, newState);
+    function changeAnalysisState(newState: PipelineStatus, activeRowsParam?: Analysis[]) {
+        const rows = activeRowsParam || activeRows;
```

Comment:
I think you can use a default argument here.
```suggestion
    function changeAnalysisState(newState: PipelineStatus, rows = activeRows) {
```